### DISCUSSION TITLE
请问如何在s.yml中配置阿里云上的SSL证书？

### DISCUSSION THREAD

**Discussion by ron-liu** [Serverless-Devs/Serverless-Devs#400] [Q&A] [answered]
我能找到的描述s.yml配置的，只有这份[文档](https://help.aliyun.com/document_detail/295899.html#h3-url-27)中，其中似乎是支持自己提供证书内容的。
![image](https://user-images.githubusercontent.com/1534952/149771525-24aa9d02-e0a4-42fa-8e66-0fb42ac726a1.png)

但是，如果我是要配置我在阿里云中产生的证书，我应该如何配置？

**Comment by heimanba** [comment, marked as answer]
也是一样的配置，直接下载证书内容即可

**Reply by ron-liu** [reply]
希望以后能支持仅填写个ARN就好。

**Comment by DevDengChao** [comment]
之前我在 fc-deploy 仓库提过一个建议 https://github.com/devsapp/fc-deploy/issues/131 , 跟这个差不多, 你可以订阅一下, 等实现好了就会收到通知.
